velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2847,3 +2847,11 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
   VELOX_ASSERT_THROW(
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
+
+TEST_F(DateTimeFunctionsTest, currentTimestampTest) {
```

Comment:
Still have some problem with how to test this function since it always gets the current timestamp

Reply:
The test you wrote to compare against `now` sounds reasonable.